### ISSUE TITLE
CS8033: The assembly Analyzers.dll does not contain any analyzers.

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn-analyzers#4917]
Should not warn for https://github.com/GuOrg/Gu.Analyzers/tree/f25670f14ee5046b76cb2941bba4aff90cdb7257

**Comment by JohanLarsson** [comment]
Where do I find the category?

I think `[assembly: SuppressMessage("??", "CS8033", Justification = "Analyzer getting it wrong.")]` is my best bet.

Adding `dotnet_diagnostic.CS8033.severity = none` to `.editorconfig` does nothing.
`<NoWarn>$(NoWarn);CS8033</NoWarn>` causes suppressions in `.gitignore` to be ignored.

**Comment by JohanLarsson** [comment]
Did some digging:

No warning
```xml
<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" />
```

Warning:
```xml
<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.9.0" />
```

**Comment by Youssef1313** [comment]
Should this be moved to roslyn? @mavasani @jaredpar
